src/evaluations/tests/analyzer_test.py

```diff
@@ -254,5 +256,45 @@ def test_get_analysis_results(self):
       self.fail('The number of estimable sets and stats is not correct.')
 
 
+class FrequencyEstimatorEvaluationAnalyzerTest(absltest.TestCase):
+
+  def test_convert_raw_df_to_long_format(self):
+    df = pd.DataFrame({
+        analyzer.SKETCH_ESTIMATOR_NAME: ['some_sketch'] * 4,
+        analyzer.SCENARIO_NAME: ['some_scenario'] * 4,
+        simulator.RUN_INDEX: [0, 0, 1, 1],
+        simulator.NUM_SETS: [1, 2, 1, 2],
+        simulator.TRUE_CARDINALITY_BASENAME + '1': [10, 20, 10, 20],
+        simulator.TRUE_CARDINALITY_BASENAME + '2': [5, 10, 5, 10],
+        simulator.ESTIMATED_CARDINALITY_BASENAME + '1': [11, 21, 12, 22],
+        simulator.ESTIMATED_CARDINALITY_BASENAME + '2': [4, 9, 3, 8],
```

Comment:
here '1', '2' refer to the frequency cap, correct?

Reply:
They refer to the n+ reach. For example, '1' refers to the 1+ reach, and '2' refers to the 2+ reach (i.e., number of ids who are seen at least twice in the campaign). And the maximum `n` can be controlled by the config.